### PR TITLE
Add the whole team and remove YLD

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

File: maintainers.txt [hunk 1]
```diff
@@ -1,8 +1,6 @@
 p.hooper@elifesciences.org
 h.drury@elifesciences.org
 c.donkin@elifesciences.org
-jennifer.spencer@yld.io
-hamant@yld.io
-javier.yenes@yld.io
-semra.shukrieva@yld.io
-#xpub-tech
```
Comment (giorgiosironi) [inline maintainers.txt]: removing the Slack channel?

Comment (diversemix) [reply maintainers.txt]: Ah I thought this was a rougue comment - oops

**Review by giorgiosironi — APPROVED** [review]

File: maintainers.txt (hunk 1 above)
Comment (thewilkybarkid) [reply maintainers.txt]: Could the format be made a bit clearer here, maybe `elifesciences.slack.com#xpub-tech`?

Comment (diversemix) [reply maintainers.txt]: Ah yes - does that format work too? Does look clearer :)

Comment (giorgiosironi) [reply maintainers.txt]: It doesn't work, but it can be made to work if everyone agrees that should be the format

Comment (diversemix) [reply maintainers.txt]: I can't imagine there won't be another person confused by the leading `#` ... this is what confused me. Although once you know its for slack it makes sense. I like @thewilkybarkid suggestion as that actually works in the browser too :)

Comment (giorgiosironi) [reply maintainers.txt]: https://github.com/elifesciences/elife-jenkins-workflow-libs/pull/16

**Pull request merged**